src/apportioned-emissions/hourly/hourly-apportioned-emissions.controller.ts

```diff
@@ -68,96 +71,96 @@ export class HourlyApportionedEmissionsController {
-  // @NotFoundResponse()
-  // @ApiQueryMultiSelect()
-  // @ApiProgramQuery()
-  // streamEmissionsFacilityAggregation(
-  //   @Req() req: Request,
-  //   @Query() params: HourlyApportionedEmissionsParamsDTO,
-  // ): Promise<StreamableFile> {
-  //   return this.service.streamEmissionsFacilityAggregation(req, params);
-  // }
+  @Get('by-facility')
+  @ApiOkResponse({
+    description:
+      'Streams Hourly Apportioned Emissions data per filter criteria aggregated by facility',
+    content: {
+      'application/json': {
+        schema: {
+          $ref: getSchemaPath(HourlyApportionedEmissionsFacilityAggregationDTO),
+        },
+      },
+      'text/csv': {
+        schema: {
+          type: 'string',
+          example: fieldMappings.emissions.hourly.data.aggregation.facility
+            .map(i => i.label)
+            .join(','),
+        },
+      },
+    },
+  })
+  @BadRequestResponse()
+  @NotFoundResponse()
+  @ApiQueryEmissionsMultiSelect()
+  @ApiProgramQuery()
+  streamEmissionsFacilityAggregation(
+    @Req() req: Request,
+    @Query() params: StreamHourlyApportionedEmissionsParamsDTO,
+  ): Promise<StreamableFile> {
+    return this.service.streamEmissionsFacilityAggregation(req, params);
+  }
 
-  // @Get('by-state')
-  // @ApiOkResponse({
-  //   description:
-  //     'Streams Hourly Apportioned Emissions data per filter criteria aggregated by state',
-  //   content: {
-  //     'application/json': {
-  //       schema: {
-  //         $ref: getSchemaPath(HourlyApportionedEmissionsStateAggregationDTO),
-  //       },
-  //     },
-  //     'text/csv': {
-  //       schema: {
-  //         type: 'string',
-  //         example: fieldMappings.emissions.hourly.data.aggregation.state
-  //           .map(i => i.label)
-  //           .join(','),
-  //       },
-  //     },
-  //   },
-  // })
-  // @BadRequestResponse()
-  // @NotFoundResponse()
-  // @ApiQueryMultiSelect()
-  // @ApiProgramQuery()
-  // streamEmissionsStateAggregation(
-  //   @Req() req: Request,
-  //   @Query() params: HourlyApportionedEmissionsParamsDTO,
-  // ): Promise<StreamableFile> {
-  //   return this.service.streamEmissionsStateAggregation(req, params);
-  // }
+  @Get('by-state')
+  @ApiOkResponse({
+    description:
+      'Streams Hourly Apportioned Emissions data per filter criteria aggregated by state',
+    content: {
+      'application/json': {
+        schema: {
+          $ref: getSchemaPath(HourlyApportionedEmissionsStateAggregationDTO),
+        },
+      },
+      'text/csv': {
+        schema: {
+          type: 'string',
+          example: fieldMappings.emissions.hourly.data.aggregation.state
+            .map(i => i.label)
+            .join(','),
+        },
+      },
+    },
+  })
+  @BadRequestResponse()
+  @NotFoundResponse()
+  @ApiQueryEmissionsMultiSelect()
+  @ApiProgramQuery()
+  streamEmissionsStateAggregation(
+    @Req() req: Request,
+    @Query() params: StreamHourlyApportionedEmissionsParamsDTO,
```

Comment:
Since aggregation endpoints don't have the exclude parameter we want to use HourlyApportionedEmissionsParamsDTO